src/RMPI/pointtopoint.jl

```diff
@@ -59,11 +59,11 @@ end
 Recieve from rank with `id` and move recieved values to `target`.
 """
 function receive!(target, s::MPIPointToPoint{P}, id) where P
-    status = MPI.Probe(id, 0, s.comm)
+    status = MPI.Probe(s.comm, MPI.Status; source=id, tag=0)
     count = MPI.Get_count(status, s.datatype)
     resize!(recvbuff(s), count)
     rb = recvbuff(s)
-    MPI.Recv!(MPI.Buffer(rb, length(rb), s.datatype), id, 0, s.comm)
+    MPI.Recv!(MPI.Buffer(rb, length(rb), s.datatype), s.comm; source=id, tag=0)
```

Comment:
Also related to changes in `MPI.jl`?

Reply:
Yes, doing it the old way triggers deprecation warnings.